mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -45,14 +45,18 @@
 
 CONTENT_TYPE_CSV = "text/csv"
 CONTENT_TYPE_JSON = "application/json"
-CONTENT_TYPE_JSON_RECORDS_ORIENTED = "application/json; format=pandas-records"
```

Comment:
Why is this change necessary?

Reply:
It did not work locally for me ---- looks like the content type was truncated by the semi-colon.  So if I send something like
`curl -X POST localhost:5000/invocations -H "Content-Type: application/json; format=pandas-records" -d"@json.txt"` it actually treated if as `"Content-Type: application/json"`